srcpkgs/yabar/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'yabar'
+pkgname=yabar
+version=0.4.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args="VERSION=$version"
+hostmakedepends="pkg-config glib-devel asciidoc"
+makedepends="libxcb-devel pango-devel libglib-devel cairo-devel
```

Comment:
`makedepends` could be minimized to `alsa-lib-devel gdk-pixbuf-devel libconfig-devel pango-devel
xcb-util-wm-devel`

Reply:
Thanks! Updated. :+1: